src/NexusMods.App.UI/Pages/LoadoutPage/ICollectionLoadoutViewModel.cs

```diff
@@ -0,0 +1,40 @@
+using Avalonia.Media.Imaging;
+using NexusMods.Abstractions.NexusWebApi.Types;
+using NexusMods.App.UI.WorkspaceSystem;
+using R3;
+
+namespace NexusMods.App.UI.Pages.LoadoutPage;
+
+public interface ICollectionLoadoutViewModel : IPageViewModelInterface
+{
+    LoadoutTreeDataGridAdapter Adapter { get; }
+
+    /// <summary>
+    /// Gets whether the collection is local or remote.
```

Comment:
So for now there is only the Default collection that is local?
Would a writable copy of a remote collection be local?

Is there any case where `IsReadOnly` and `IsLocalCollection` wouldn't match?

Reply:
I don't know yet, we're not there yet.

Reply:
Fair answer